zebrad/src/components/mempool/storage.rs

```diff
@@ -136,33 +143,36 @@ impl Storage {
             return Err(MempoolError::InMempool);
         }
 
-        // Then, we try to insert into the pool. If this fails the transaction is rejected.
+        // Try to insert into the pool. If this fails the transaction is rejected.
```

Comment:
Not needed comment change
```suggestion
        // Then, we try to insert into the pool. If this fails the transaction is rejected.
```